package.json

```diff
@@ -12,7 +12,8 @@
     "build": "npm run fix-olm-dependency && rm -rf ./build && tsc",
     "lint": "npm run fix-olm-dependency && tsc --noEmit && eslint . --ext .js,.ts --quiet",
     "lint:fix": "tsc --noEmit && eslint . --ext .js,.ts --fix --quiet",
-    "prepare": "husky"
+    "prepare": "husky",
+    "dev": "ts-node-dev --inspect --respawn --exit-child --watch src --transpile-only src/index.ts"
```

Comment:
Do we really need a different command to start the server in development mode. If this one does the thing better then we might need to remove the start script. I don't see a big point for keeping two separate commands for same purpose.